RosettaCore/src/test/java/com/hubspot/rosetta/RosettaBinderTest.java

```diff
@@ -118,20 +120,17 @@ public void itUsesServiceLoaderToDiscoverModules() {
     bean.setId(50L);
     bean.setName("test");
 
-    assertThat(bind(bean)).isEqualTo(map("id_value", "50", "name_value", "test"));
-    assertThat(bindWithPrefix("prefix", bean)).isEqualTo(map("prefix.id_value", "50", "prefix.name_value", "test"));
-  }
-
-  private static Map<String, Object> map(String... strings) {
```

Comment:
Because `bind` returns `Map<String, Object>`, and type variance is something that the Java type system doesn't implement, so `Map<Object, Object>` is not a superclass of `Map<String, Object>` so the comparisons fail.